### DISCUSSION TITLE
Still actively developed?

### DISCUSSION THREAD

**Discussion by HeiziND** [hyperdiv/hyperdiv#75] [General]
Hi,
I'm interested to dig into and learn more about this project, but can't but notice that there hasn't been any commits in months. Would it be possible to elaborate on whether this is still being actively worked on ?

**Comment by richstokes** [comment]
I was wondering this too. Seems like a really interesting framework, but don't want to get too far with it if it's likely to be abandoned

**Comment by mariusnita** [comment]
There has definitely been a lull here, as other parts of my life took over, but I plan to keep going with Hyperdiv.

**Reply by richstokes** [reply]
Exciting to hear. Honestly a really cool project and think it has a lot of potential

**Reply by AERCR3** [reply]
是的，这种及时渲染的web框架，在敲代码的时候就像VSCode上的彩虹屁一样令人愉悦，更确切的说这个框架我感觉更像是Markdown的Python的Web增强版，希望能成为下一个MarkdownPlusPlus，类似C与Cpp
